cli/runner.py

```diff
@@ -34,35 +37,34 @@ def start(factories: dict[str, workflow.executor.WorkflowStepFactory] = MappingP
     resource_manager = workflow.manager.ResourceManager.init(logger, args.engine, args.database, env_config)
     logger.info("Using: " + ",".join(f"{k}={v}" for k, v in vars(args).items()))
     try:
-        logger.info(f"Activating batch with config from '{args.batch_config}'")
         start_time = time.time()
-        # load batch config as json string
-        batch_config_json = workflow.utils.read_config(args.batch_config)
         # create engine if it doesn't exist
         resource_manager.add_engine(args.engine_size)
-        # Skip infrastructure setup during recovery
-        if not args.recover and not args.recover_step:
+        if args.action == "init":
```

Comment:
@AndrewNepogoda maybe it's better to create **RUN** and **INIT** constants in the `workflow.constants`?